examples/lock-app/silabs/efr32/BUILD.gn

```diff
@@ -61,6 +62,11 @@ efr32_sdk("sdk") {
     include_dirs += [ "${examples_plat_dir}/wf200" ]
   }
 
+  if (chip_enable_ble_rs911x) {
```

Comment:
This goes against @jmartinez-silabs cleanup no ?

Reply:
Best solution currently. Will have to refactore and move some files in they created in that include dir